cmd/tasks.go

```diff
@@ -4,34 +4,51 @@ import (
 	"crypto"
 	"fmt"
 
-	"github.com/anchore/syft/syft/artifact"
-
-	"github.com/anchore/syft/syft/sbom"
-
+	"github.com/anchore/syft/internal/config"
 	"github.com/anchore/syft/syft"
+	"github.com/anchore/syft/syft/artifact"
 	"github.com/anchore/syft/syft/file"
+	"github.com/anchore/syft/syft/sbom"
 	"github.com/anchore/syft/syft/source"
 )
 
-type powerUserTask func(*sbom.Artifacts, *source.Source) ([]artifact.Relationship, error)
+type task func(*sbom.Artifacts, *source.Source) ([]artifact.Relationship, error)
 
-func powerUserTasks() ([]powerUserTask, error) {
-	var tasks []powerUserTask
+func tasks(appConfig *config.Application) ([]task, error) {
+	var tasks []task
 
-	generators := []func() (powerUserTask, error){
+	generators := []func() (task, error){
```

Comment:
Short answer: generators create tasks. When I wrote this initially I probably should have gone with something like `generateCatalogPackagesTask` instead of `catalogPackagesTask` (rinse and repeat for each function that creates a task [generator]).

Reply:
Ah, I gotcha! I like that improvement.